scipy/linalg/tests/test_lapack.py

```diff
@@ -3413,3 +3413,45 @@ def test_tgsyl(dtype, trans, ijob):
                         err_msg='lhs1 and rhs1 do not match')
         assert_allclose(lhs2, rhs2, atol=atol, rtol=0.,
                         err_msg='lhs2 and rhs2 do not match')
+
+
+@pytest.mark.parametrize('dtype', DTYPES)
+@pytest.mark.parametrize('lower', (0, 1))
+def test_sytrs(dtype, lower):
```

Comment:
Would you mind if I were to consolidate the tests?

Reply:
Sure no problem, but note that sytrs is 4 flavors and hetrs is 2 flavors. Symmetric complex matrices are skew-hermitian hence csytrs and zsytrs and chetrs and zhetrs are different things. Just in case that was what you had in mind.

Reply:
Yup. Still, the changes were pretty modest, and incidentally it made the the `hetrs` tests deterministic. (`np.random.rand` was used in the `hetrs` tests but not `sytrs`.) I'll go ahead and merge if tests are still green.